src/bin/pgcopydb/extensions.c

```diff
@@ -173,10 +174,12 @@ copydb_copy_ext_table(PGSQL *src, PGSQL *dst, char *qname, char *condition)
 		.srcWhereClause = condition,
 		.dstQname = qname,
 		.dstAttrList = "",
-		.bytesTransmitted = 0
+		.bytesTransmitted = 0,
+		.bytesTransmittedBeforeSavingProgress = 0,
+		.lastSavingTimeMs = time(NULL),
```

Comment:
This looks like it should be maintained in the callback's context, not here.

Reply:
Done. what about `bytesTransmitted`? It does not seem to belong here either.

Reply:
You're right. I believe it was a trade-off that made sense with the previous code. Now that we have a separate context to handle this dynamic aspect, it's time to do the proper refactoring here.